File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        possibleSol = {}
        
        for i, j in enumerate(nums):
            sub = target - j
            if sub in possibleSol:
                return [possibleSol[sub], i]
            possibleSol[j] = i;
            
        return

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
   
        unordered_map<int,int> possibleSol;
        vector<int> answer;
        
        //Loop through number vectors
        for(int i = 0; i < nums.size(); i++){
            int sub = target - nums[i];
             
            if(possibleSol.find(sub) != possibleSol.end()){
                answer.push_back(possibleSol[sub]); 
                answer.push_back(i);  
            }else{
                possibleSol.insert({nums[i], i});
            }
            
        }
        //Returning answer
        return answer;
    }
};